client-react/src/pages/app/functions/function/function-log/FunctionLogAppInsightsDataLoader.tsx

```diff
@@ -132,7 +141,7 @@ const FunctionLogAppInsightsDataLoader: React.FC<FunctionLogAppInsightsDataLoade
     liveLogsSessionId?: string
   ) => {
     quickPulseQueryLayer
-      .queryDetails(tokenComponent.token, false, '', liveLogsSessionId, functionsRuntimeVersion)
+      .queryDetails(tokenComponent.token, false, '', useNewFunctionLogsApi(), liveLogsSessionId)
```

Comment:
If using `useMemo` instead of `useCallback`:
```suggestion
      .queryDetails(tokenComponent.token, false, '', useNewFunctionLogsApi, liveLogsSessionId)
```